### PR TITLE
Add datepicker feature

### PR DESCRIPTION
**Pull request by rastorc3v**

### CODE REVIEW

File: src/app/shared/components/datepicker/datepicker.component.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@Component({
+  selector: 'app-datepicker',
+  templateUrl: './datepicker.component.html',
+  styleUrls: ['./datepicker.component.scss']
+})
+export class DatepickerComponent {
+  range = new FormGroup({
+    start: new FormControl(),
+    end: new FormControl()
+  });
```
Comment (rmurtazin) [inline src/app/shared/components/datepicker/datepicker.component.ts:14]: add spaces between methods

Comment (rastorc3v) [reply src/app/shared/components/datepicker/datepicker.component.ts]: Ready, thanks

File: src/app/shared/components/datepicker/datepicker.component.ts [hunk 2]
```diff
@@ -10,10 +10,13 @@ import { IDateRange } from '@interfaces/dateRange.interface';
 export class DatepickerComponent {
   range = new FormGroup({
```
Comment (rmurtazin) [inline src/app/shared/components/datepicker/datepicker.component.ts:11]: modificator and type

File: src/app/app.module.ts [hunk 3]
```diff
@@ -44,6 +44,11 @@ import { MarkerIconComponent } from './core/components/map/marker-icon/marker-ic
 import { ClusterIconComponent } from './core/components/map/cluster-icon/cluster-icon.component';
 import { BonusListComponent } from './core/components/bonus-list/bonus-list.component';
 import { MatExpansionModule } from '@angular/material/expansion';
+import { DatepickerComponent } from './shared/components/datepicker/datepicker.component';
+import { MatDatepickerModule } from '@angular/material/datepicker';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatNativeDateModule } from '@angular/material/core';
+import {MatInputModule} from '@angular/material/input';
```
Comment (rmurtazin) [inline src/app/app.module.ts:51]: Spaces between word and brackets, if you run prettier it would become in general style

**Review by rmurtazin — APPROVED** [review]

File: src/app/app.module.ts (hunk 3 above)
Comment (rastorc3v) [reply src/app/app.module.ts:51]: thanks, i've not used to use it yet.

**Pull request merged**